### DISCUSSION TITLE
Build Error in Vercel

### DISCUSSION THREAD

**Discussion by ReaseySereiNeath** [vercel/community#523] [Help] [answered]
<html>
<body>

I'm sorry I'm new here. I don't know what's wrong can u help me with this?

Cloning gitlab.com/g4251/gk-tourism-project (Branch: main, Commit: 41bbd0f)
--
19:13:40.165 | Cloning completed: 1.088s
19:13:40.273 | Installing build runtime...
19:13:43.064 | Build runtime installed: 2.791s
19:13:44.296 | No Build Cache available
19:13:44.498 | Installing dependencies...
19:13:44.743 | yarn install v1.22.17
19:13:44.804 | warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
19:13:44.807 | [1/4] Resolving packages...
19:13:44.937 | [2/4] Fetching packages...
19:14:11.526 | [3/4] Linking dependencies...
19:14:11.528 | warning "@emotion/styled > @emotion/babel-plugin@11.7.2" has unmet peer dependency "@babel/core@^7.0.0".
19:14:11.528 | warning "@emotion/styled > @emotion/babel-plugin > @babel/plugin-syntax-jsx@7.16.7" has unmet peer dependency "@babel/core@^7.0.0-0".
19:14:34.162 | [4/4] Building fresh packages...
19:14:34.176 | Done in 49.44s.
19:14:34.200 | Detected Next.js version: 12.0.10
19:14:34.203 | Running "yarn run build"
19:14:34.477 | yarn run v1.22.17
19:14:34.509 | $ next build
19:14:35.426 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
19:14:35.427 | This information is used to shape Next.js' roadmap and prioritize features.
19:14:35.427 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
19:14:35.427 | https://nextjs.org/telemetry
19:14:35.427 |  
19:14:35.594 | info  - Checking validity of types...
19:14:35.719 | warn  - No ESLint configuration detected. Run next lint to begin setup
19:14:35.722 | info  - Creating an optimized production build...
19:14:37.559 |  
19:14:37.559 | warn - You have enabled the JIT engine which is currently in preview.
19:14:37.560 | warn - Preview features are not covered by semver, may introduce breaking changes, and can change at any time.
19:15:00.056 | Failed to compile.
19:15:00.056 |  
19:15:00.057 | ./components/Header.js
19:15:00.057 | Error: error: Expected corresponding JSX closing tag for <header>
19:15:00.057 |  
19:15:00.057 | \|
19:15:00.057 | 180 \|           </div>
19:15:00.057 | \|           ^^^^^^
19:15:00.057 |  
19:15:00.057 | Caused by:
19:15:00.057 | 0: failed to process js file
19:15:00.057 | 1: Syntax Error
19:15:00.057 |  
19:15:00.058 | Import trace for requested module:
19:15:00.058 | ./pages/search.js
19:15:00.058 |  
19:15:00.058 |  
19:15:00.058 | > Build failed because of webpack errors
19:15:00.099 | error Command failed with exit code 1.
19:15:00.100 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
19:15:00.110 | Error: Command "yarn run build" exited with 1

</body>
</html>

**Comment by swarnava** [comment, marked as answer]
Hi,



It happens when we do not close a html tag.

Make sure all the html tags are closed.

For example:

❌ 

```
<input id="icon_prefix" type="text" class="validate">
```

✅ 

```
<input id="icon_prefix" type="text" class="validate" />
```

Try temporarily removing piece of code until you find which html tag closing is missing.
